multicast/send.py

```
@@ -277,27 +277,27 @@ def setupArgs(cls, parser):
 		"""
 		if parser is not None:  # pragma: no branch
 			parser.add_argument(
-				"""--port""",
+				"--port",
 				type=int,
 				default=multicast._MCAST_DEFAULT_PORT  # skipcq: PYL-W0212 - module ok
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)